src/components/HamburgerMenu.tsx

```diff
@@ -90,12 +91,16 @@ const HamburgerMenu = () => {
           // Therefore, position the HamburgerMenu at top: 1px so that the sidebar is not accidentally opened on tab change.
           top: positionFixedStyles.top + 1,
         }}
-        {...fastClick(() => {
+        {...fastClick(e => {
           // TODO: Why does the sidebar not open with fastClick or onTouchEnd without a setTimeout?
           // onClick does not have the same problem
           setTimeout(() => {
             dispatch(toggleSidebar({}))
           }, 10)
+          // The Editables need to ignore the click event and its corresponding blur, fired after all of the touch events.
+          if (isTouch && isSafari()) {
+            e.preventDefault()
+          }
```

Comment:
Can you remind me why this is needed? What happens if it is omitted?

It's often helpful in an inline comment for a special case to include an "Otherwise..." clause to make it super clear.

Thanks!

Reply:
If it's omitted, Safari fires a click event when the sidebar is closed, despite not firing one when the sidebar is opened. It's not necessary for this implementation that we've settled on, but it will block future efforts at focus restoration. Up to you whether it stays in or gets removed for now. I like it because it makes the events symmetrical, and there are fewer of them.